pages/multi/[address]/index.tsx

```diff
@@ -47,17 +45,22 @@ const multipage = (_props: Props) => {
   const fetchMultisig = async (address: string) => {
     setAccountError(null);
     try {
-      const client = await StargateClient.connect(state!.chain.nodeAddress!);
-      const tempHoldings = await client.getBalance(address, state!.chain.denom!);
+      assert(state.chain.nodeAddress, "Node address missing");
+      const client = await StargateClient.connect(state.chain.nodeAddress);
+      assert(state.chain.denom, "denom missing");
+      const tempHoldings = await client.getBalance(address, state.chain.denom);
       const tempAccountOnChain = await getMultisigAccount(address, client);
       setHoldings(tempHoldings);
       setAccountOnChain(tempAccountOnChain);
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
     } catch (error: any) {
       setAccountError(error.message);
       console.log("Account error:", error);
     }
   };
 
+  assert(state.chain.addressPrefix, "address prefix missing");
```

Comment:
Feels like this assert should be in the `fetchMultisig` function.  Any reason it's top level in this component?